bibliography-sage.bib

```diff
@@ -3919,7 +3919,7 @@ @article{OkadoEtAl2016
   note         = {https://arxiv.org/abs/1603.08121},
   number       = {2},
   pages        = {341--401},
-  title        = {Type {$D_n^{(1)}$} rigged configuration bijection},
+  title        = {Type $D_n^{(1)}$ rigged configuration bijection},
```

Comment:
So, I guess the curly brackets you removed meant "keep this as it is".

![Screenshot from 2022-02-25 17-32-19](https://user-images.githubusercontent.com/207405/155751861-47230546-7d2a-4987-b9ac-ca95b19ce357.png)